### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by johnie261**
### Hello there
 ## hello

### i did the following activities per the requirements given:

- Added the missing JSX elements.
- Styled the app to match zeplin design.
- Follow GitFlow

### CODE REVIEW

File: src/components/book.js [hunk 1]
```diff
@@ -18,14 +18,36 @@ const Book = ({ itemId, title, author }) => {
...
+      </div>
+      <div className="update">
+        <div className="horizontal-line" />
+        <div className="update-progress">
+          <p className="current">CURRENT CHAPTER</p>
+          <p className="chapter">Chapter 3: &quot;A lesson learned &quot;</p>
+          <button type="button" className="btn-chapter">UPDATE PROGRESS</button>
         </div>
       </div>
     </section>
```
Comment (tresorsawasawa) [inline src/components/book.js]: - [ ] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
Screenshot from 2023-04-07 02-58-24

### Your design
Screenshot from 2023-04-07 02-57-38

File: src/components/inputBook.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ const InputBook = () => {
   return (
     <>
       <form className="form">
+        <div className="vertical-line" />
         <h1 className="heading">ADD NEW BOOK</h1>
         <div className="inputs">
           <input type="text" value={newBook} onChange={handleChange} placeholder="Book title" className="input" required />
```
Comment (tresorsawasawa) [inline src/components/inputBook.js:42]: - [ ] As  required for the project, kindly make sure your styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) e.g.: Text colors, Fonts(size, weight, family,...), background-colors, alignment, spscing(psdding & mergin),...
### Template design
Screenshot from 2023-04-07 03-07-49
### Your design
Screenshot from 2023-04-07 03-08-29

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @johnie261 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/inputBook.js (hunk 2 above)
Comment (johnie261) [reply src/components/inputBook.js:42]: Ok, let me do the changes

**Review by TadesseDev — APPROVED** [review]
### Hi @johnie261 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**